ballerina/googlesheets_client.bal

```diff
@@ -268,17 +267,21 @@ public isolated client class GoogleSheetsClient {
                         return <persist:Error>error(value.message());
                     }
                     values.push(value);
-                } else {
+                }
+
+                else {
```

Comment:
```suggestion
                } else {
```

Reply:
Fixed in [6af274a](https://github.com/ballerina-platform/module-ballerinax-persist.googlesheets/pull/15/commits/6af274ac3b3c8e6595db3232d142746015cd4ef0)